src/System.Private.Xml/src/project.json

```diff
@@ -39,6 +39,7 @@
         "System.Runtime": "4.3.0-beta-24522-03",
         "System.Runtime.Extensions": "4.3.0-beta-24522-03",
         "System.Runtime.InteropServices": "4.3.0-beta-24522-03",
+        "System.Security.Permissions": "4.3.0-beta-24522-03",
```

Comment:
We should not depend on System.Security.Permissions here. Can you please remove whatever needs that dependency.